src/pages/ReimbursementAccount/EnableStep.js

```diff
@@ -1,56 +1,134 @@
+import _ from 'underscore';
 import React from 'react';
-import {Image, View} from 'react-native';
+import PropTypes from 'prop-types';
+import {View, Image} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
 import styles from '../../styles/styles';
 import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
-import {navigateToConciergeChat} from '../../libs/actions/Report';
 import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
 import Navigation from '../../libs/Navigation/Navigation';
 import Text from '../../components/Text';
-import CONST from '../../CONST';
-import TextLink from '../../components/TextLink';
 import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import {ChatBubble} from '../../components/Icon/Expensicons';
+import MenuItem from '../../components/MenuItem';
+import getBankIcon from '../../components/Icon/BankIcons';
+import {getPaymentMethods} from '../../libs/actions/PaymentMethods';
+import FullScreenLoadingIndicator from '../../components/FullscreenLoadingIndicator';
+import bankAccountPropTypes from '../../components/bankAccountPropTypes';
+import {navigateToConciergeChat} from '../../libs/actions/Report';
+import confettiPop from '../../../assets/images/confetti-pop.gif';
+import Icon from '../../components/Icon';
+import WorkspaceSection from '../workspace/WorkspaceSection';
+import {ConciergeBlue} from '../../components/Icon/Illustrations';
 
 const propTypes = {
+    /** Are we loading payment methods? */
+    isLoadingPaymentMethods: PropTypes.bool,
+
+    /** Array of bank account objects */
+    bankAccountList: PropTypes.arrayOf(bankAccountPropTypes),
+
     ...withLocalizePropTypes,
 };
 
-const EnableStep = ({translate}) => {
-    const verifyingUrl = `${CONST.CLOUDFRONT_URL}/images/icons/emptystates/emptystate_reviewing.gif`;
-    return (
-        <View style={[styles.flex1, styles.justifyContentBetween]}>
-            <HeaderWithCloseButton
-                title={translate('validationStep.headerTitle')}
-                onCloseButtonPress={Navigation.dismissModal}
-            />
-            <View style={[styles.flex1]}>
-                <Image
-                    source={{uri: verifyingUrl}}
-                    style={[styles.workspaceInviteWelcome]}
-                    resizeMode="center"
+const defaultProps = {
+    isLoadingPaymentMethods: true,
+    bankAccountList: [],
+};
+
+class EnableStep extends React.Component {
+    componentDidMount() {
+        getPaymentMethods();
```

Comment:
NAB but I _think_ we could front-load payment methods by calling `getPaymentMethods` in `AuthScreens` instead of in `EnableStep`